### DISCUSSION TITLE
Welcome to web Discussions!

### DISCUSSION THREAD

**Discussion by pluja** [monerogarden/web#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about yourself!

**Comment by k4r4b3y** [comment]
Turkish speaking monerobro here.  Currently I have a signal group chat for turkanons about monero: https://signal.group/#CjQKICUccbfPqIGYux906JmjTTY9TIplefrNHCNB-rNdOB7kEhDwqp9rYNPC4ccS0nROndI6
